src/plugins/home/public/application/components/homepage/hero_section/get_started.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { i18n } from '@osd/i18n';
+import { FormattedMessage } from '@osd/i18n/react';
+import {
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiTitle,
+  EuiImage,
+  EuiLink,
+  EuiIcon,
+} from '@elastic/eui';
+import { HeroSection } from './hero_section';
+import illustration from '../../../../assets/illustration.svg';
+import { getServices } from '../../../opensearch_dashboards_services';
+import logo from '../../../../assets/logos/chat.svg';
+
+export const GetStartedSection: React.FC<{ olly?: boolean }> = ({ olly = true }) => {
+  const services = getServices();
+  const getUrl = services.application.getUrlForApp;
+  const navigate = services.application.navigateToApp;
+  const prompts = services.homeConfig.prompts;
+  type Prompt = typeof prompts extends Array<infer T> ? T : never;
+
+  const OLLY_DESCRIPTION = (
+    <FormattedMessage
+      id="home.getStarted.ollyDescription"
+      defaultMessage="Automatically generate complex queries using simple natural language questions. AI assisted summaraies help you navigate and understand your log data."
+    />
+  );
+
+  const PLAYGROUND_DESCRIPTION = (
+    <FormattedMessage
+      id="home.getStarted.playgroundDescription"
+      defaultMessage="Automatically generate complex queries using simple conversational prompts. AI assisted summary helps you navigate and understand errors from your logs.{br}{br}You will be redirected to the observability playground where you will need to login. All the {terms} of the playground still apply."
+      values={{
+        br: <br />,
+        terms: (
+          <EuiLink href="https://opensearch.org/terms-of-use/">
+            <FormattedMessage id="home.getStarted.terms" defaultMessage="terms of use" />
+          </EuiLink>
+        ),
+      }}
+    />
+  );
+
+  const description = olly ? OLLY_DESCRIPTION : PLAYGROUND_DESCRIPTION;
+
+  const OLLY_ACTION_BUTTON = (
```

Comment:
i think this can be abstracted enough to be configurable. which would feed into a dynamic solution